stacks-signer/src/client/stacks_client.rs

```diff
@@ -797,7 +801,11 @@ impl StacksClient {
         let body =
             json!({"sender": self.stacks_address.to_string(), "arguments": args}).to_string();
         let path = self.read_only_path(contract_addr, contract_name, function_name);
-        let timer = crate::monitoring::new_rpc_call_timer(&path, &self.http_origin);
+        let timer_label = format!(
+            "{}/v2/contracts/call-read/:principal/{contract_name}/{function_name}",
```

Comment:
If we can reasonably say `{contract_name}` and `{function_name}` isn't going to change often or will have fewer than ~100 permutations, this should be fine.

Reply:
As of right now, `contract_name` is only either `signers` or `signers-voting` (and `signers-voting` isn't even used in the actual running signer), and then `function_name` is only 1 value, and once `signers-voting` is used (in "v1" of the signer, down the road), it's another ~5 functions.